components/org.wso2.carbon.identity.oauth.common/src/main/java/org/wso2/carbon/identity/oauth/common/OAuthConstants.java

```diff
@@ -247,6 +247,9 @@ public static SubjectType fromValue(String text) {
             "SupportedIDTokenEncryptionMethod";
     public static final String REQUEST_OBJECT_ENCRYPTION_METHOD = "OAuth.OpenIDConnect." +
             "SupportedRequestObjectEncryptionMethods.SupportedRequestObjectEncryptionMethod";
+
+    public static final String ENABLE_TLS_CERT_TOKEN_BINDING = "OAuth.OpenIDConnect." +
```

Comment:
We have already added a constant for this **ENABLE_TLS_CERT_BOUND_ACCESS_TOKENS_VIA_BINDING_TYPE**

Reply:
changed